docs/reference/plugins/py-splashscreen.md

```diff
@@ -0,0 +1,65 @@
+
+# &lt;py-splashscreen&gt;
+
+This is one of the core plugins in PyScript, which is active by default. The splashscreen is the first thing you see when you open a page with Pyscript while it is loading itself and all the necessary resources.
+
+## Configuration
+
+You can control how `<py-splashscreen>` behaves by setting the value of the  `splashscreen` configuration in your `<py-config>`.
+
+
+| parameter   | default | description |
+|-------------|---------|-------------|
+| `autoclose` | `true`  | Whether to close the splashscreen automatically when the page is ready or not |
+| `enabled` | `true`    | Whether to show the splashscreen or not |
```

Comment:
Is the attribute called `enabled` or `disabled`?

Reply:
Oops yeah `disabled` thanks! Do you think `enabled` would be better? Just wondering

Reply:
Ooh yes, I think `enabled` (defaults to _true_) is easier to understand than `disabled` (defaults to _false_). Avoids a bit of a doble-negative.

Reply:
Awesome updated 😄